rtengine/color.cc

```diff
@@ -1910,6 +1911,147 @@ void Color::Lch2Luv(float c, float h, float &u, float &v)
     v = c * sincosval.y;
 }
 
+void Color::primaries_to_xyz (double p[6], double Wx, double Wz, double *pxyz)
+{
+	//calculate Xr, Xg, Xb, Yr, Yb, Tg, Zr,Zg Zb
+	double Wy = 1.0;
+    double Xr = p[0] / p[1];
```

Comment:
I am correct to assume that the values of `p` can be user-specified? If so, are values of 0 acceptable? If so, we need guards here to prevent DIV/0.

Reply:
Well seen, good catch.

Thank you.
I just push a commit to solve this case.

Jacques